hazelcast/src/main/resources/hazelcast-default.xml

```diff
@@ -69,12 +69,6 @@
                 <zones>us-east1-b,us-east1-c</zones>
             </gcp>
             <azure enabled="false">
-                <client-id>CLIENT_ID</client-id>
-                <client-secret>CLIENT_SECRET</client-secret>
-                <tenant-id>TENANT_ID</tenant-id>
-                <subscription-id>SUB_ID</subscription-id>
-                <cluster-id>HZLCAST001</cluster-id>
-                <group-name>RESOURCE-GROUP-NAME</group-name>
```

Comment:
Are these removed on purpose? The rest of the discovery plugins have default values set explicitly.

Reply:
Yes, we don't need any properties by default on member config if the user sets up the identites (See more detail here: https://github.com/hazelcast/hazelcast-azure#basic-configuration). For full config, I added all available properties. Please see `hazelcast-full-example.xml`.